### PR TITLE
Anamorphic fixes to video widget

### PR DESCRIPTION
**Pull request by jonoomph**
A few more fixes for anamorphic video profile support.

### CODE REVIEW

File: src/windows/video_widget.py [hunk 1]
```diff
@@ -77,28 +77,20 @@ def changed(self, action):
...
+                    action.values.get("num", 1),
+                    action.values.get("den", 1))
                 log.info(
                     "Update: Set video widget pixel aspect ratio to: %s",
                     self.pixel_ratio.ToFloat())
 
-            # Update max size (to size of video preview viewport)
-            if display_ratio_changed or pixel_ratio_changed:
-                timeline = get_app().window.timeline_sync.timeline
-                timeline.SetMaxSize(
```
Comment (jonoomph) [inline src/windows/video_widget.py:97]: This call does not seem needed anymore

File: src/windows/video_widget.py [hunk 2]
```diff
@@ -77,28 +77,20 @@ def changed(self, action):
             if action.type == "load" and action.values.get("pixel_ratio"):
                 pixel_ratio_changed = True
                 self.pixel_ratio = openshot.Fraction(
-                    action.values.get("pixel_ratio").get("num", 16),
```
Comment (jonoomph) [inline src/windows/video_widget.py:80]: pixel_ratios should default to 1/1, if not found

File: src/windows/video_widget.py [hunk 3]
```diff
@@ -77,28 +77,20 @@ def changed(self, action):
...
-                    action.values.get("pixel_ratio").get("den", 9))
+                    action.values.get("pixel_ratio").get("num", 1),
+                    action.values.get("pixel_ratio").get("den", 1))
                 log.info(
                     "Set video widget pixel aspect ratio to: %s",
                     self.pixel_ratio.ToFloat())
             elif action.key and action.key[0] == "pixel_ratio":
                 pixel_ratio_changed = True
```
Comment (JacksonRG) [inline src/windows/video_widget.py]: I searched the project, and the only use of `pixel_ratio_changed` and `display_ratio_changed` is in the if statement that got deleted. So this and the initialization of those two bools should probably be taken out.

**Review by JacksonRG — APPROVED** [review]
2 variables that aren't being used anymore. Best get rid of those for future clarity.

Other than that, LG

File: src/windows/video_widget.py (hunk 3 above)
Comment (jonoomph) [reply src/windows/video_widget.py]: Good catch! Removing them now

**Pull request merged**